tools/travis/test.sh

```diff
@@ -26,5 +26,4 @@ WHISKDIR="$ROOTDIR/../openwhisk"
 
 export OPENWHISK_HOME=$WHISKDIR
 cd ${ROOTDIR}
-TERM=dumb ./gradlew :tests:checkScalafmtAll
```

Comment:
Why did you drop this? Does scalafmt run automatically now?

Reply:
Hello, yes, exactly.

Reply:
Fixed